docs/NOW-spec.md

```diff
@@ -0,0 +1,1601 @@
+
+# NOW-PROTO
+
+<style>
+    .byte-layout {
+        width: 100%;
+        table-layout: fixed;
+    }
+    .byte-layout th {
+        colspan: 1;
+        text-align: center;
+        vertical-align: bottom;
+    }
+    .byte-layout td {
+        text-align: center;
+    }
+</style>
+
+
+[[_TOC_]]
```

Comment:
Likewise, this have no effect on GitHub, and there is already a table of contents by clicking on the three dots at the top right of the markdown page
![image](https://github.com/user-attachments/assets/a53b714f-8aec-427e-ab43-33c872dab3b7)